README.md

```diff
@@ -2,53 +2,48 @@
 
 > v1.4 now allows the counting to be done on the homepage too!
 
-# Introduction
+## Demo
 
-![](screenshots/wordcount.gif)
+<img width="80%" src="screenshots/wordcount.gif" />
 
-# Usage
+## Usage
 
-Use `/wordcount`. It will then count the number of words below this parent block. This supports an unlimited number of nested blocks.
+| command | autocompletes to | what it does |
+| :- | :- | :- |
+| `/wordcount` | `{{renderer :wordcount_RANDOM-ID}}` | count the number of **words** below this parent block |
+| `/wordcount char` | `{{renderer :wordcountchar_RANDOM-ID}}` | count the number of **characters** below this parent block |
```

Comment:
This is the primary change in this PR. Everything else is just tiny cosmetic things to make it a little more readable.